src/components/html-wrapper.tsx

```
@@ -10,6 +10,13 @@ export default function HtmlWrapper({ children }: { children: ReactNode }) {
   const { isRTL } = useDirection();
   return (
     <html lang={locale} suppressHydrationWarning dir={isRTL ? 'rtl' : 'ltr'}>
+      <script
+        defer
+        suppressHydrationWarning
+        dangerouslySetInnerHTML={{
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)